packages/lodestar/src/api/impl/validator/index.ts

```diff
@@ -166,6 +171,20 @@ export function getValidatorApi({chain, config, logger, metrics, network, sync}:
       );
   }
 
+  function computeProposersForEpoch(state: CachedBeaconStateAllForks, epoch: Epoch): ValidatorIndex[] {
+    const effectiveBalanceIncrementsWithLen: EffectiveBalanceIncrements = getEffectiveBalanceIncrementsWithLen(
+      state.validators.length
+    );
+
+    for (let i = 0; i < state.validators.length; i++) {
```

Comment:
This should be available in EpochContext, we don't have to recalculate again

Reply:
Updated